File: app/src/main/java/rbsoftware/friendstagram/PictureViewHolder.java
package rbsoftware.friendstagram;

import android.support.v7.widget.RecyclerView;
import android.view.View;

import com.facebook.drawee.view.SimpleDraweeView;

/**
 * Created by silver_android on 09/11/16.
 */

public class PictureViewHolder extends RecyclerView.ViewHolder {
    SimpleDraweeView image;

    PictureViewHolder(View itemView) {
        super(itemView);
        image = (SimpleDraweeView) itemView.findViewById(R.id.image);
    }
}

File: app/src/main/java/rbsoftware/friendstagram/ui/viewholder/PictureViewHolder.kt
package rbsoftware.friendstagram.ui.viewholder

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import rbsoftware.friendstagram.R

/**
 * Created by silver_android on 09/11/16.
 */

class PictureViewHolder(containerView: View?) : RecyclerView.ViewHolder(containerView) {
    val image: ImageView? = containerView?.findViewById(R.id.image)
}
